tests/components/test_population.py

```diff
@@ -8,11 +8,11 @@
     {
         "household_id": [1] * 5,
         "date_of_birth": [
-            pd.Timestamp("1987-03-01 00:00:00"),
-            pd.Timestamp("2023-05-11 00:00:00"),
-            pd.Timestamp("2026-04-01 00:00:00"),
-            pd.Timestamp("2025-11-21 00:00:00"),
-            pd.Timestamp("1992-07-25 00:00:00"),
+            pd.Timestamp("1985-01-01 00:00:00"),
```

Comment:
Can we define constants for `DOB_BEFORE_SIM_START` and `DOB_AFTER_SIM_START` or something? So that it is explicit what the important difference is between these two dates?